Aardvark/ARKLogTableViewController.m

```diff
@@ -402,16 +404,18 @@ - (void)_reloadLogs;
 {
     [self.logStore retrieveAllLogMessagesWithCompletionHandler:^(NSArray *logMessages) {
         self.logMessages = [self _logMessagesWithMinuteSeparators:logMessages];
-        [self _reloadFilteredLogs];
+        [self _reloadFilteredLogs:false];
```

Comment:
Reads as "do not reload filtered logs." :-/ It might be more straightforward to add a new `searchStringForFilteredLogs` ivar that is always updated when `filteredLogs` is, so that the `isSubset` calculation can be done entirely within `_reloadFilteredLogs` (and is not dependent on the current state of the `searchController.searchBar`).